src/pages/workspaces/workspace/workflows/FailureRerunner.js

```diff
@@ -42,7 +42,7 @@ export const rerunFailures = async ({ namespace, name, submissionId, configNames
       _.map('workflowEntity')
     )(workflows)
 
-    const newSetName = `${configName}-resubmission-${new Date().toISOString().slice(0, -5).replace(/:/g, '-')}`
+    const newSetName = `${configName}-resubmission-${new Date().toISOString().slice(0, -5).replace(/[^\w]/g, '-')}` // colons in date, periods in wf name
```

Comment:
```suggestion
    const newSetName = `${configName}-resubmission-${new Date().toISOString().slice(0, -5)}`.replace(/[^\w]/g, '-') // colons in date, periods in wf name
```

Reply:
You could make the `.replace(...)` part a function in `job-common.js`, like:
```js
const sanitizeWorkflowName = name => name.replace(/[^\w]/g, '-')
```
or even
```js
const buildWorkflowName = base => `${base}_{new Date().toISOString().slice(0, -5)}`.replace(/[^\w]/g, '-')
```